modules/dind/dind.go

```diff
@@ -0,0 +1,112 @@
+package dind
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+var (
+	// containerPorts {
+	defaultDockerDaemonPort = "2375/tcp"
+)
+
+// DinDContainer represents the Docker in Docker container type used in the module
+type DinDContainer struct {
```

Comment:
suggestion: use Container here to avoid the stutter.

@mdelapenya I'm guessing we need to update the template for this?